libffi-rs/src/middle/types.rs

```diff
@@ -102,6 +102,7 @@ where
     let new = ffi_type_array_create_empty(size);
     for (i, element) in elements.enumerate() {
         *new.add(i) = *element.0;
+        forget(element);
```

Comment:
@xtldr `forget` itself isn't imported, instead `std::mem` as a whole is imported. This means you have to use `mem::forget(element)` here.

Reply:
@YorickPeterse Yes of course. I shouldn't use the online editor to create patches without testing, even for one liners. I have fixed it now.